inbox/events/timezones.py

```diff
@@ -86,31 +101,49 @@
     "SA Pacific Standard Time": "America/Bogota",
     "SA Western Standard Time": "America/La_Paz",
     "SE Asia Standard Time": "Asia/Bangkok",
+    "Saint Pierre Standard Time": "America/Miquelon",
+    "Sakhalin Standard Time": "Asia/Sakhalin",
     "Samoa Standard Time": "Pacific/Apia",
+    "Sao Tome Standard Time": "Africa/Sao_Tome",
+    "Saratov Standard Time": "Europe/Saratov",
     "Singapore Standard Time": "Asia/Singapore",
     "South Africa Standard Time": "Africa/Johannesburg",
+    "South Sudan Standard Time": "Africa/Juba",
     "Sri Lanka Standard Time": "Asia/Colombo",
+    "Sudan Standard Time": "Africa/Khartoum",
     "Syria Standard Time": "Asia/Damascus",
     "Taipei Standard Time": "Asia/Taipei",
     "Tasmania Standard Time": "Australia/Hobart",
+    "Tocantins Standard Time": "America/Araguaina",
     "Tokyo Standard Time": "Asia/Tokyo",
+    "Tomsk Standard Time": "Asia/Tomsk",
     "Tonga Standard Time": "Pacific/Tongatapu",
+    "Transbaikal Standard Time": "Asia/Chita",
     "Turkey Standard Time": "Europe/Istanbul",
+    "Turks And Caicos Standard Time": "America/Grand_Turk",
     "US Eastern Standard Time": "America/Indianapolis",
     "US Mountain Standard Time": "America/Phoenix",
-    "UTC": "Etc/GMT",
+    "UTC": "Etc/UTC",
     "UTC+12": "Etc/GMT-12",
+    "UTC+13": "Etc/GMT-13",
     "UTC-02": "Etc/GMT+2",
+    "UTC-08": "Etc/GMT+8",
+    "UTC-09": "Etc/GMT+9",
     "UTC-11": "Etc/GMT+11",
     "Ulaanbaatar Standard Time": "Asia/Ulaanbaatar",
     "Venezuela Standard Time": "America/Caracas",
     "Vladivostok Standard Time": "Asia/Vladivostok",
+    "Volgograd Standard Time": "Europe/Volgograd",
     "W. Australia Standard Time": "Australia/Perth",
     "W. Central Africa Standard Time": "Africa/Lagos",
     "W. Europe Standard Time": "Europe/Berlin",
+    "W. Mongolia Standard Time": "Asia/Hovd",
     "West Asia Standard Time": "Asia/Tashkent",
+    "West Bank Standard Time": "Asia/Hebron",
     "West Pacific Standard Time": "Pacific/Port_Moresby",
     "Yakutsk Standard Time": "Asia/Yakutsk",
+    "Yukon Standard Time": "America/Whitehorse",
+    "tzone://Microsoft/Utc": "Etc/UTC",
```

Comment:
Saw this one in the wild as well: https://gitlab.gnome.org/GNOME/evolution-ews/-/issues/85